### ISSUE TITLE
support "lowercase" subscribed containers

### ISSUE THREAD

**Issue by ZeeD** [lovasoa/marshmallow_dataclass#136]
since python 3.9 you can use `list[foo]` instead of `typing.List[foo]` in annotations, as you can see in the [pep 585](https://www.python.org/dev/peps/pep-0585/).

unfortunately I see errors instead using this library. Example code:

```python
from marshmallow_dataclass import class_schema
from typing import List
from dataclasses import dataclass

@dataclass
class C1:
    lst: List[int]

@dataclass
class C2:
    lst: list[int]

class_schema(C1)().loads('{"lst":[1,2,3]}')
class_schema(C2)().loads('{"lst":[1,2,3]}')
```

I get no error from `C1`, but `AttributeError: type object 'list' has no attribute '__dataclass_fields__'` using `C2` (and a bit of `During handling of the above exception, another exception occurred:` related exceptions).

Using python `3.9.1`, `marshmallow==3.11.1`, `marshmallow-dataclass==8.4.1`

**Comment by lovasoa** [comment]
Blocked by https://github.com/ilevkivskyi/typing_inspect/issues/65

**Comment by martinitus** [comment]
Ping. Typing_inspect released `0.7.1` where they attempted to support builtin generics :-)
Btw: awesome library! ❤️

**Comment by lovasoa** [comment]
Yes, it should work now! Anyone interested in making a pull request?
